stdlib/public/Concurrency/DiscardingTaskGroup.swift

```diff
@@ -184,159 +184,6 @@ public struct DiscardingTaskGroup {
     let _: Void? = try await _taskGroupWaitAll(group: _group, bodyError: nil)
   }
 
-// Clone the task-creation routines in Embedded Swift so that we don't
-// introduce an implicit use of `any Actor`.
-#if !$Embedded
```

Comment:
Are you sure that the `#if !$Embedded` branch is the one to remove? It contains the `@isolated(any)` additions (introduced by commit a86b76a3e9ebe352e8d250927c211445ac692c26).